docs/pages/concepts/offloading/pod-offloading.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Pod offloading
+weight: 3
+---
+
+Differently from a traditional kubelet, which starts the containers of a Pod on the designated Node, the Liqo virtual kubelet maps each operation to a corresponding *twin* Pod object in the remote cluster for actual execution.
+At the same time, it takes care of the automatic propagation of *remote status changes* to the corresponding local Pod, allowing for complete observability from the local cluster.
+Finally, advanced operations, such as metrics and logs retrieval, as well as interactive command execution inside remote containers, are transparently supported, to comply with standard troubleshooting operations.
+
+### Remote Pod resiliency
+
+Due to the permanent link between the two twin Pod representations (in the local and in the remote cluster), different behaviors can occur when deleting an offloaded Pod:
+
+- The Pod is deleted in the **local cluster** (intentionally or by eviction): the remote one has to be immediately deleted. In fact, the local cluster is the owner of the pod, therefore whatever modification to the Pod is set, this has to be reflected in the remote Pod.
+- The Pod is deleted in the **remote cluster** (intentionally or by eviction): not only the local Pod must not be deleted, but the remote Pod has to be recreated as soon as possible. This recovery process shall not require the local cluster intervention, to reduce the overhead and ensure service continuity even in case of temporary connectivity loss between the two control planes.
+
```

Comment:
I would add here that when a remote *node* disappear, we do the same actions as if the remote pod was deleted.